get_aqi.py

```diff
@@ -119,6 +152,25 @@ def get_aqi_color(aqi_level, country):
         elif aqi_level == "very high":
             # Red : 101 < aqi
             return "255 0 0"
+    elif country == "US":
+        if aqi_level == "Good":
+            # Green : 0 < aqi <= 50
+            return "121 227 71"
```

Comment:
I tested on my device colors that are more accurate :

```suggestion
            return "0 255 0"
```